src/ui/server/controllers/insurance/business/broker/index.test.ts

```diff
@@ -48,6 +49,12 @@ describe('controllers/insurance/business/broker', () => {
     });
   });
 
+  describe('FIELD_IDS', () => {
+    it('should have the correct FIELD_IDS', () => {
+      expect(FIELD_IDS).toEqual([USING_BROKER, NAME, ADDRESS_LINE_1, ADDRESS_LINE_2, TOWN, COUNTY, POSTCODE, EMAIL]);
```

Comment:
Not a blocker, can be done in the next PR, we can use `expected` const here so it's easier to read